src/doc/stdmetadata.rst

```diff
@@ -164,6 +164,17 @@ Color information
     window that are not overlapping the pixel data window.  If not supplied,
     the default is black (0 in all channels).
 
+.. option:: "CICP" : int8[4]
+
+    The CICP color space information, as defined by
+    `ITU-T H.273 <https://www.itu.int/rec/T-REC-H.273>`_.  This is an array
+    of four 8-bit integers, with the following meanings:
```

Comment:
```suggestion
    of four integers, with the following meanings:
```

Reply:
It may be stored in the files as 4 uint8's, but through the OIIO APIs it should be passed as the much more common int type, no?